src/main/java/core/basesyntax/services/impl/Writer.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax.services.impl;
+
+import core.basesyntax.Storage;
+import core.basesyntax.model.Fruit;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Writer {
+    private static final String STOCK_FILE = "src/main/resources/stock.csv";
```

Comment:
it's better to accept file path as an method argument

Reply:
fixed it